github-sample/.github/workflows/logicapp_container_deploy.yml

```
@@ -0,0 +1,62 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/azure/functions-action
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: (container) Build and Deploy Logic App. 
+
+env:
+  AZURE_LOGICAPP_PACKAGE_PATH: '.'
+  DOTNET_VERSION: '3.1.301'    # set this to the dotnet version to use
+  RG_LA: 'your-la-rg'
+  RG_CON: 'your-con-rg'
+  LA_NAME: 'your-la-name'
+  Cont_Image: 'your-cont-image'
+  Cont_Tag: 'your-tag'
+  acrUsername: 'your-acr-name'
```

Let's reduce the duplication
```suggestion
  acrUsername: 'your-acr-name'
  IMAGE_URI: '${{ env.acrUsername }}.azurecr.io/${{ env.Cont_Image }}:${{ env.Cont_Tag }}'
```

---

Resolved in latest commit. Thanks.